### DISCUSSION TITLE
"Find hosts and Pods running Crowdstrike using the DaemonSet and Sidecar method."

### DISCUSSION THREAD

**Discussion by afroz1** [CrowdStrike/falconpy#1067] [Ideas]
We have Kubernetes running in nearly all of our environments, with us managing the worker nodes, and in a few environments, the worker nodes are managed by a vendor. We have deployed Node Sensor(DaemonSet) and Container Sensor(Sidecar) through the Operator on that environment.

Do we have a script that can be used to fetch information about hosts (worker nodes) and Pods from our Crowdstrike EU portal and then share the status report with the users?

**Comment by jshcodes** [comment]
Hi @afroz1 -

Not yet, but this is on the horizon! 

A large number of kubernetes / container related API operations have been released over the past few weeks.  These will be implemented in the `1.3.4` version of FalconPy. (These updates are in flight now. ✈️)

**Comment by afroz1** [comment]
Hi @jshcodes,
Thanks for your prompt response. It's indeed a well-thought-out plan and very beneficial, especially considering our extensive Kubernetes environment with various vendors. Do we have a timeline for when the script will be ready?

**Reply by jshcodes** [reply]
Hi @afroz1 -

`1.3.4` is slated to hit the production index sometime end of this month / early next.  Development for samples demonstrating these new API operations will begin sometime after that. (Considering holidays, etc. probably next quarter.) I've added this item to our sample request list. 😄

**Reply by afroz1** [reply]
Hi jscodes,

Any update on this request? I am curious to know if the script is ready to find the hosts and Pods running CrowdStrike using the DaemonSet and Sidecar method.

**Reply by jshcodes** [reply]
Hi @afroz1 -

We are currently working on implementing these operations within the SDK now. (The container operations were bumped to version `1.3.5`.) The development for the sample requested above will occur after these operations are available. 😄

**Reply by afroz1** [reply]
Hi jshcodes,

Any good news for me ?

**Reply by jshcodes** [reply]
HI @afroz1 - 

The new container operations have been released and documented.  We will begin working on a sample that leverages [ReadNodeCombined](https://falconpy.io/Service-Collections/Kubernetes-Protection.html#readnodecombined) and [ReadPodCombined](https://falconpy.io/Service-Collections/Kubernetes-Protection.html#readpodcombined). 😄

**Reply by afroz1** [reply]
Good News for my Organisation. Please let us know once the script get ready.
